Process/PullRequests.md

```diff
@@ -43,6 +43,7 @@ Each PR will:
           Example: “/cc @nmonterroso”.
        2. If the PR contains production SQL changes the @SQLReview tag should be
           used to notify the SQL review team.
+       3. If the PR cointain multiple tasks it can be useful to provide them in form of [Task List](https://github.com/blog/1825-task-lists-in-all-markdown-documents)
```

Comment:
It might be worth mentioning that this could also potentially help with sharing work across multiple team members